cypress/test/Roadmap/RoadmapItem.cy.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { mount } from '@cypress/react';
+import RoadmapItem from '../../../components/roadmap/RoadmapItem'
+
+describe('RoadmapItem', () => {
```

Comment:
`RoadmapItem` should contain a case for clicking the Item and it opens a modal containing necessary details for each item.

Reply:
sure